src/BotPlutusInterface/CardanoCLI.hs

```diff
@@ -251,13 +254,7 @@ submitTx pabConf tx =
           , networkOpt pabConf
           ]
       )
-      ( ( \out ->
```

Comment:
I realised that the callCommand will throw an IOErr exception when fails, so parsing the output is unnecessary, it's enough to know that the exitcode is 0